cmd/tendermint/commands/reset_priv_validator.go

```diff
@@ -28,25 +30,38 @@ func init() {
 // XXX: this is totally unsafe.
 // it's only suitable for testnets.
 func resetAll(cmd *cobra.Command, args []string) {
-	resetPrivValidator(cmd, args)
-	os.RemoveAll(config.GetString("db_dir"))
-	os.Remove(config.GetString("cs_wal_file"))
+	ResetAll(config, log)
 }
 
 // XXX: this is totally unsafe.
 // it's only suitable for testnets.
 func resetPrivValidator(cmd *cobra.Command, args []string) {
+	resetPrivValidatorLocal(config, log)
+}
+
+func ResetAll(c *viper.Viper, l log15.Logger) {
+	resetPrivValidatorLocal(c, l)
+	dataDir := c.GetString("db_dir")
+	walFile := c.GetString("cs_wal_file")
```

Comment:
no I mean the walFile is now a folder within the dataDir, so if you wipe the dataDir, the walFile will be wiped too